### ISSUE TITLE
With known displacement field

### ISSUE THREAD

**Issue by stephahn** [sfepy/sfepy#615]
Hi
My question is about running a simulation with a known displacement field. I have multiples 2D images with deformations occurring (~15-20 frames) and the flow field using a dense optical flow. 

Is it possible to use Sfepy to compute the stiffness/strain, inside multiple hypothesis contexts (with and without damping, forced oscillation, etc...)?

thanks in advance

**Comment by rc** [comment]
Hi @stephahn, I am not familiar with the problem you are trying to solve, but yes, it is possible to compute stress, strain and other quantities for a given displacement field. IIRC there is not a ready example for this.

**Comment by stephahn** [comment]
Thanks for the answer. Do you plan to make a demo?

**Comment by rc** [comment]
You mean something like "read mesh and displacements from a file and compute strains"?

**Comment by stephahn** [comment]
yes, exactly. actually I have a mesh file and a dense flow field array.

**Comment by rc** [comment]
Could you attach here a small example mesh and the corresponding flow field data?
